src/main/java/com/groupesan/project/java/scrumsimulator/mainpackage/ui/widgets/BlockerWidget.java

```diff
@@ -106,12 +140,11 @@ private void resolveBlocker() {
                 JOptionPane.YES_NO_OPTION);
 
         if (confirmation == JOptionPane.YES_OPTION) {
-            BlockerStore.getInstance(simulationId).removeBlocker(blocker.getId());
+            BlockerStore.getInstance().removeBlocker(blocker.getId());
```

Comment:
This will cause error after merging, as the simulationId is required to store blocker in simulation.json